examples/lines_bars_and_markers/masked_demo.py

```diff
@@ -1,30 +1,46 @@
-'''
-===========
-Masked Demo
-===========
+"""
+==============================
+Plotting masked and NaN values
+==============================
 
-Plot lines with points masked out.
+Sometimes you need to plot data with missing values.
 
-This would typically be used with gappy data, to
-break the line at the data gaps.
-'''
+One possibility is to remove the undesired points from the data. In this case,
+a plotted line throughout the data is contignuous. It connects all given data
```

Comment:
I think you mean “continuous”.  And probably “through” instead of “throughout”. though you could argue either word you used is ok.  I’d actually rephrase the whole thing as:

“One possibility is to simply remove undesired data points.  The line plotted through the remaining data will be continuous, and not indicate where the missing data is located.

If it is useful to have gaps in the line where the data is missing, then the undesired points can be indicated using a `masked array`_ or by setting their values to ``np.NaN``.  No maker will be drawn where either *x* or *y* are masked, and if plotting with a line it will be broken there. “

Reply:
@jklymak Thanks, I've used your proposal. - Only using plain NaN instead of `np.NaN` because you can also set it via `float('nan')` or other ways and I don't want to be specific on the way here.